modules/apps/adaptive-media/adaptive-media-web/src/main/resources/META-INF/resources/adaptive_media/js/EditAdaptiveMediaConfig.es.js

```diff
@@ -129,9 +126,7 @@ class EditAdaptiveMediaConfig extends PortletBase {
 	 * @param {KeyboardEvent} event The keyboard event.
 	 */
 	handleKeyDown_(event) {
-		const code = event.keyCode || event.charCode;
-
-		if (this.validInputKeyCodes_.indexOf(code) == -1) {
+		if (VALID_INPUT_KEYS.indexOf(event.key) == -1) {
```

Comment:
Prefer `===` over `==`.

As a matter of habit, if you want to look up something by value, you should be looking it up in a data structure that has `O(1)` lookup.

So, maybe you should change `VALID_INPUT_KEYS` to be `new Set([...])`, and then you can do:

```js
if (VALID_INPUT_KEYS.has(event.key)) {
    // ....
}
```

(I know you are just carrying forward the existing `indexOf`, but may as well improve things while we're at it as per the Boy Scout Rule.)

Reply:
I was caught again in the law of the good scout.

Fixing!